openlibrary/plugins/ol_infobase.py

```diff
@@ -479,7 +479,7 @@ def process_edition_doc(self, doc):
         return doc
 
     def normalize_edition_title(self, title):
-        if isinstance(title, str):
+        if isinstance(title, bytes):
```

Comment:
If this is still waiting on my review, I don't think this will or should ever trigger. Title should always be a str at this point and should not need to be decoded.

Reply:
Great timing @hornc !
It is on my todo list for this week to merge this PR.  Which of the following changes would you recommend?

___Current code:___
```python
        if isinstance(title, bytes):
            title = title.decode('utf-8', 'ignore')

        if not isinstance(title, six.text_type):
            return ""
```
1.
```python
        if not isinstance(title, six.text_type):  # or six.string_types
            raise ValueError("title: %s must be a string.")
```
-- OR --
2.
```python
        if not isinstance(title, six.text_type):  # or six.string_types
            return ""
```

Reply:
@cclauss I think 1 is better.